steps/createwinpeplainuefiisofromwinpeamd64onwindowsworker/script.txt

```diff
@@ -0,0 +1,18 @@
+echo a
+    
+if {kickstartedBootLoaderIsUefi}==true (

```

Comment:
This should be 
```suggestion
if (${{kickstartedBootLoaderIsUefi}}) (

```

Reply:
Hi @jarrodchesney , I tried that
```
if (${{kickstartedBootLoaderIsUefi}}) (
```
and got this error because it's a batch script I think.
![image](https://github.com/Attune-Automation/Automate-Windows-Installation-with-autounattend/assets/14309370/9a285407-ffd0-4cc0-9440-350c4804dfd6)

but this works:
```
if {kickstartedBootLoaderIsUefi}==true (
```
![image](https://github.com/Attune-Automation/Automate-Windows-Installation-with-autounattend/assets/14309370/609cba67-9fa5-4bcd-a95f-b05a61821f17)